packages/scripts/src/application/application.ts

```diff
@@ -76,15 +61,9 @@ export class Application {
         resolvedEnvironments: ReturnType<typeof getResolvedEnvironments>;
     }> {
         const opts = defaults(options, buildDefaults);
-        const { config: _config, path: configPath } = await this.getEngineConfig();
-        const config: EngineConfig = defaults(_config, { externalFeatureDefinitions: [] as IExternalDefinition[] });
+        const { config: _config } = await this.getEngineConfig();
+        const config = defaults(_config, buildDefaults);
 
-        if (opts.external && !opts.featureName) {
-            throw new Error('You must specify a feature name when building a feature in external mode');
-        }
-        if (config.require) {
```

Comment:
this is important. engine.config.js -> require are hooks that we use to install `@stylable/node/r` and others